code/game/objects/items/mail.dm

```diff
@@ -113,7 +113,7 @@
 		var/datum/mind/recipient = recipient_ref.resolve()
 		// If the recipient's mind has gone, then anyone can open their mail
 		// whether a mind can actually be qdel'd is an exercise for the reader
-		if(recipient && recipient != user?.mind)
+		if(recipient && recipient.name != user.real_name)
```

Comment:
what if the recipient changes their name with genetics? does that change their mind name?

Reply:
prolly, tho admittedly its funny to see genetic commit identity fraud for a mails

Reply:
```suggestion
		if(recipient && recipient.name != user.real_name && !considered_afk(recipient))
```

Since you changed the mind code to compare name, don't forget to still check if the recipient went afk.

Reply:
Is it needed tho? Since opening it is an active action the user must not be afk to begin with right?

Reply:
> prolly, tho admittedly its funny to see genetic commit identity fraud for a mails

The recipient. as in the person who the mail is intended for.

Reply:
> Is it needed tho? Since opening it is an active action the user must not be afk to begin with right?

You are checking if the recipient is afk, not the user.